src/Storage/Commands.Storage.Test/Commands.Storage.Test.csproj

```diff
@@ -88,17 +88,11 @@
       <HintPath>..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory, Version=2.28.3.860, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I am not sure you cna remove these

Reply:
I don't see any negative impact by removing it.